framework/decode/referenced_resource_table.cpp

```diff
@@ -104,7 +115,7 @@ void ReferencedResourceTable::AddResourceToContainer(format::HandleId container_
     }
 }
 
-void ReferencedResourceTable::AddResourceToUser(format::HandleId user_id, format::HandleId resource_id)
+void ReferencedResourceTable::AddResourceToUser(format::HandleId user_id, format::HandleId resource_id, bool print)
```

Comment:
Did you mean to remove `print` here and in other locations?

Reply:
Right, missed that